apps/studio/styles/main.scss

```diff
@@ -198,10 +198,10 @@ input[type='number'] {
   display: none;
 }
 
-code {
-  // use supabase-ui code style
-  @apply text-code;
-}
+// code {
```

Comment:
👁️  ? 👁️

Reply:
yea this can be removed, we shouldn't be doing this imo.
was causing havoc in code blocks

Reply:
yeap let's remove if its not used